src/components/Modals/WorldPermissionsModal/ModelTabs/Layouts/WorldPermissionsAvatarWithInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { AvatarFace } from 'decentraland-ui/dist/components/AvatarFace/AvatarFace'
+import LoadingText from 'decentraland-ui/dist/components/Loader/LoadingText'
+import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard'
+import { getResumedAddress } from '../../utils'
+import { WorldPermissionsAvatarWithInfoProps } from './WorldPermissionsAvatarWithInfo.types'
+
+import styles from './WorldPermissionsAvatarWithInfo.module.css'
+
+export default React.memo(function WorldPermissionsAvatarWithInfo(props: WorldPermissionsAvatarWithInfoProps) {
+  const { wallet, profiles, loading } = props
+
+  if (loading || !profiles || !wallet) {
+    return (
+      <div className={styles.avatar}>
+        <AvatarFace size="small" inline />
+        <LoadingText className={styles.loadingtext} type="p" size="medium"></LoadingText>
+      </div>
+    )
+  }
+
+  return (
+    <div className={styles.avatar}>
+      <AvatarFace className={styles.avatarface} avatar={profiles[wallet]} size="small" inline />
```

Comment:
We have a `Profile` componen in the `decentraland-dapps` repo, which is an enhanced version of the `Profile` component that comes from the UI, which automatically fetches the profiles.

Reply:
Making 100 fetch at the same time is not the better solution. I'll stick with this one if you don't mind